### DISCUSSION TITLE
Feat: A hosted debugger

### DISCUSSION THREAD

**Discussion by Complexlity** [wevm/frog-archived#44] [Ideas]
The dev ui is really cool (the best frames debugger ui I've seen). It would be cool if the same was hosted on [https://frog.fm/](https://frog.fm/) so other's can debug frames by insertting a URL (also with the option to log in with warpcast)

**Comment by tmm** [comment]
Thanks! This is something that we are looking into. Transaction support and fixing initial issues are the top priority for us at the moment.

One of the reasons the Frog Devtools is so powerful is that Frog Frames get injected with extra [context data](https://github.com/wevm/frog/blob/main/src/frog-base.tsx#L387) in development, allowing them to render additional debugging information that isn't available in non-Frog Frames. This means that you get the best experience when using Frog with the Frog Devtools (and that non-Frog Frames would not have access to all the same devtools features).

**Comment by Complexlity** [comment]
Oh I get it. I would say it (the hosted one) doesn't really need to be that powerful. As long as it functions as good as the official one, it should work well

**Comment by vrypan** [comment]
I'm building a frame with Python, designed to run on AppEngine. I would love to be able to use frog in the following way:
1. Just run `npx something` (I really don't want to touch node/js/ts etc) to run the dev ui.
2. I get a form to enter my (local or remote) frame URL to display and debug it.

**Comment by PirosB3** [comment]
> I'm building a frame with Python, designed to run on AppEngine. I would love to be able to use frog in the following way:
> 
> 1. Just run `npx something` (I really don't want to touch node/js/ts etc) to run the dev ui.
> 
> 2. I get a form to enter my (local or remote) frame URL to display and debug it.

Same here, I'm also using Python so the existing JS stack won't work. Would definitely prefer Frog to work on localhost

**Comment by tmm** [comment]
Added support for this in frog@0.7

https://warpcast.com/awkweb/0x77ce28e1
